.misc/check_unsupported.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set +e
+
+# Enable capturing the non-zero exit status of coverage instead of tee
+set -o pipefail
+
+coverage run setup.py install | tee setup.log
+
+retval=$?
+
+if [[ $retval != 4 ]]; then
+  echo "Unexpected error code $?"
+  if [[ $retval == 0 ]]; then
```

Comment:
hm every error code except 4, but 0 is forbidden? I assume that's on purpose, even so I'm curious :3